File: C/URI/1020.c
#include <stdio.h>
 
int main() {
 
 int  mes, ano, restoano, restomes, nasc;
        scanf("%d", &nasc);
        restoano = 0;
        ano =0;
        mes =0;
        if(nasc >= 365){
        {
                restoano = nasc % 365;
                ano = nasc / 365;
        }
                if ( restoano >= 30 )
                {
                        restomes = restoano % 30;
                        mes = restoano / 30;
                }
        }
        else{
                restomes = nasc % 30;
                mes = nasc / 30;
        }
        printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", ano, mes, restomes);

    return 0;
}

File: JS/URI/1020.js
const input = require('fs').readFileSync('stdin', 'utf8');
const lines = input.split('\n');

let days = parseInt(lines.shift());

let years = parseInt(days/365);
days = days - (years*365);

let months = parseInt(days/30);
days = days - (months*30);

console.log(years+" ano(s)");
console.log(months+" mes(es)");
console.log(days+" dia(s)");